api/src/main/resources/workbench.yaml

```diff
@@ -750,6 +750,43 @@ paths:
           schema:
             $ref: '#/definitions/ErrorResponse'
 
+  /v1/admin/unsafe-self-bypass-access-requirement/{moduleName}:
+      post:
+        tags:
+          - profile
+        consumes:
+          - application/json
+        description: >
+          Updates the given user to bypass the request access requirement,
+          e.g. "eraCommons", or "twoFactorAuth"
```

Comment:
Agreed. I can handle the refactor of that in this PR.

Reply:
If the work has been put in already that's fine, but I'd personally argue this isn't worth the effort (we've got more important things to clean up!). This isn't causing any actual pain since we don't have external API clients. You could simply document the full set of expected strings.

Reply:
At this point I'm almost done with it, so I think I am going to continue (wasn't a ton of work, just a bit of wrestling with Swagger)

Since its tech debt associated with my work, I figure it doesn't hurt to just clean it up, if it doesn't take too long.

Reply:
Where should a Workbench dev expect to find the documentation if not here? This will show up in generated code in both the client and server models as well. I think it's a good idea to make this interface authoritative, regardless of whether or not we have external clients.

Also, we do have two sets of strings which are currently being manually kept in sync when there's an easy alternative. I agree none of this is particularly high priority.